hvat/src/hyperview/asset_api_functions.rs

```diff
@@ -343,15 +343,27 @@ pub async fn search_assets_async(
         .json::<Value>()
         .await?;
 
-    if let Some(metadata) = &resp.get("_metadata") {
-        let total = metadata["total"].as_u64().unwrap();
-        let limit = metadata["limit"].as_u64().unwrap();
-        info!("Meta Data: | Total: {} | Limit: {} |", total, limit);
-    }
+    let total = resp
+        .get("estimatedTotalHits")
+        .expect("Expected a value")
```

Comment:
suggest adding a more descriptive error. e.g. expected a value for estimated total hits variable. It would make troubleshooting errors easier.